src/modules/sentry/sentry.service.ts

```diff
@@ -29,7 +30,28 @@ export class SentryService implements OnModuleDestroy, OnApplicationShutdown {
   constructor(protected readonly configService: ConfigService) {}
 
   private isExcludedExceptions(exception: unknown): boolean {
-    return this.EXCLUDED_EXCEPTIONS.some((item) => exception instanceof item);
+    if (this.EXCLUDED_EXCEPTIONS.some((item) => exception instanceof item)) {
+      return true;
+    }
+
+    if (
+      exception instanceof QueryFailedError &&
+      exception.message.search(
+        'duplicate key value violates unique constraint'
+      ) !== -1
+    ) {
+      return true;
+    }
+
+    if (
+      exception instanceof Error &&
+      (exception.message === 'Invalid DID' ||
+        exception.message === 'Invalid did provided')
+    ) {
+      return true;
+    }
```

Comment:
Same rationale as the first comment.
```suggestion
    // This error is excluded because...
    if (
      exception instanceof Error &&
      (exception.message === 'Invalid DID' ||
        exception.message === 'Invalid did provided')
    ) {
      return true;
    }
```